### PR TITLE
Website updates

### PR DESCRIPTION
**Pull request by christianalfoni**
- Now points to packages README file
- Due to failing `babel-watch` on looking at packages folder (probably due to node_modules and no config to handle it), I had to implement own watcher for the docs files
- Improved provider docs
- Improved view install docs
- Improved general install docs to also state YARN method
- Added router docs
- Refactored factories of router to operators concept

### CODE REVIEW

**Comment by christianalfoni** [comment]
Last but not least, SWAG store link is in there :-D

**Comment by Guria** [comment]
> Refactored factories of router to operators concept

in website commits. Are you kidding? :)

File: packages/cerebral-router/README.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+  goTo('/items')
+]
+```
+
+### redirect
+*action*
+```js
+function myAction({router}) {
+  // Go to a new url, replacing current url
+  router.goTo('/items')
```
Comment (henri-hulski) [inline packages/cerebral-router/README.md]: goTo -> redirect

Comment (christianalfoni) [inline packages/cerebral-router/README.md]: Fixed :)

**Pull request merged**